Add documentation for Bang & Olufsen button Event entites (#35062)
home-assistant/home-assistant.github.io: source/_integrations/bang_olufsen.markdown

WebSocket notifications received from the device are fired as events in Home Assistant. These can be received by listening to `bang_olufsen_websocket_event` event types, where `device_id` or `serial_number` can be used to differentiate devices.

> ### Button events

Reviewer: Let's remove button from here. While I get the idea from the integrated device perspective, from the Home Assistant end naming it buttons can be confusing, as that is a different concept.

```suggestion
### Events
```

Event entities are created for each of the physical "buttons" on your device. These buttons usually have their own behaviors, so using them for automations is not always ideal.

> Available buttons:

Reviewer: ```suggestion
Available event entities:
```

- Bluetooth
- Microphone
- Next
- Play / Pause
- Favourite 1
- Favourite 2
- Favourite 3
- Favourite 4
- Previous
- Volume

> All of these buttons support the following events:

Reviewer: ```suggestion
All of these event entities support the following event types:
```
